resources/core/requirements.yaml

```diff
@@ -1,6 +1,6 @@
 dependencies:
-  - name: dex
-    condition: dex.enabled
+  # - name: dex
```

Comment:
It can be removed from here as dex is no longer part of core instead of commenting out :)

Reply:
Looks like I'm way behind the master, I will rebase ofc. shortly :)